slinker_web/src/client/scss/components/Collection.scss

```diff
@@ -0,0 +1,61 @@
+.o-slinker-collection {
+  padding: 1.2rem 1rem 1.2rem;
+  font-family: 'Ubuntu', Arial, sans-serif;
+  border-bottom: 1px solid #000;
+}
+
+.o-slinker-collectionheader {
+  
+  &> h2.o-title {
+    position: relative;
+    display: inline-block;
+    margin: 0;
+    line-height: 1.6;    
+    font-weight: normal;
+    font-size: 1rem;
+    user-select: none;
+    cursor: pointer;
+  }
+
+  &> button.o-icon {
+    display: inline-block;
+    width: 1.4rem;
+    height: 1.4rem;
+    vertical-align: top;
+    padding: 1px 6px;
+    border: none;    
+    margin-left: 0.5rem;
+    text-indent: -1000rem;
+    text-transform: none;
+    background-size: 1.2rem 1.2rem;
+    background-color:#000;          
+    outline: none;
+    cursor: pointer;
+
+    &__expand {
+      /* https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images */
+      -webkit-mask:  url('/images/expand-icon.svg') no-repeat 50% 50%;
+      mask: url('/images/expand-icon.svg') no-repeat 50% 50%;  
+
+      &:hover {
+        background-color: red;
+      }
+    }
+
+    &__collapse {
+      /* https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images */
+      -webkit-mask:  url('/images/collapse-icon.svg') no-repeat 50% 50%;
```

Comment:
doesn't work in IE 11 or Edge 14/15

Reply:
OK, I'll look for something different then